app/assets/javascripts/components/story/StoryPopover.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import Markdown from '../Markdown';
+import Popover from 'components/jquery_wrappers/Popover.js';
+
+export const StoryPopoverContent = ({ description, notes, createdAt, storyType, requestedByName }) => (
+  <div className='popover__content'>
+    <div className='popover__content__subtitle'>
+      {
+        I18n.translate('requested by user on date', {
+          user: requestedByName,
+          date: moment(createdAt).format('DD MM YYYY, h:mm a')
+        })
+      }
+
+      <div className='text-right'>
+        { I18n.translate('story.type.' + storyType) }
```

Comment:
Can we use interpolation here?

```suggestion
        { I18n.translate(`story.type.${storyType}`) }
```